hack/run-tests.sh

```diff
@@ -28,7 +28,8 @@ fi
 source ./hack/check_operator_condition.sh
 printOperatorCondition
 
-${TEST_OUT_PATH}/func-tests.test -ginkgo.v -junit-output="${TEST_OUT_PATH}/output/junit.xml" -installed-namespace="${INSTALLED_NAMESPACE}" -cdi-namespace="${INSTALLED_NAMESPACE}" "$@" "${KUBECONFIG_FLAG}"
+mkdir "${TEST_OUT_PATH}/output"
```

Comment:
Please use 
```suggestion
mkdir -p "${TEST_OUT_PATH}/output"
```
to make it more robust (no error if existing, make parent directories as needed)